compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmConstants.java

```diff
@@ -436,6 +439,8 @@ public class JvmConstants {
     public static final String OBSERVABLE_ANNOTATION = "ballerina/observe/Observable";
     public static final String DISPLAY_ANNOTATION = "display";
     public static final String RECORD_CHECKPOINT_METHOD = "recordCheckpoint";
+    public static final String MODULE_INITIALIZER_METHOD_DESC =
+            "(Lio/ballerina/runtime/internal/scheduling/Strand;)Ljava/lang/Object;";
```

Comment:
We can move this to JvmSignatures class

Reply:
Fixed in 8da28c04c1397a39613ef773b0f56c5989ce3016.